docs/integration/jenkins.md

```diff
@@ -0,0 +1,58 @@
+**jenkins.http.response.code.bad_request** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/400 status code  
+**jenkins.http.response.code.created** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/201 status code  
+**jenkins.http.response.code.forbidden** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/403 status code  
+**jenkins.http.response.code.no_content** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/204 status code  
+**jenkins.http.response.code.not_found** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/404 status code  
+**jenkins.http.response.code.not_modified** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/304 status code  
+**jenkins.http.response.code.ok** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/200 status code 
+**jenkins.http.response.code.server_error** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/500 status code 
+**jenkins.http.response.code.server_unavailable** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a HTTP/503 status code 
+**jenkins.http.response.code.other** <br>*(long counter)*    |  The count at which the Jenkins master Web UI is responding to requests with a non-informational status code that is not in the list - HTTP/200, HTTP/201, HTTP/204, HTTP/304, HTTP/400, HTTP/403, HTTP/404, HTTP/500, or HTTP/503 
+**jenkins.nodes.builds** <br>*(long counter)*      |  The count of build nodes available to Jenkins
+**jenkins.nodes.builds.time** <br>*(long counter)* *(ms)*      |  The time nodes spend for building
+**jenkins.nodes.offline** <br>*(long gauge)*      |  The count of build nodes available to Jenkins but currently off-line
+**jenkins.nodes.online** <br>*(long gauge)*      |  The count of build nodes available to Jenkins and currently on-line
+**jenkins.queue.blocked** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.buildable** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.pending** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.size** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue
+**jenkins.queue.stuck** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
```

Comment:
This seems wrong...

Reply:
Sharp eyes, copied from this https://plugins.jenkins.io/metrics/ seems they has wrond values too ))
Changed.